common/law_groups/00_laws.txt

```diff
@@ -25,15 +25,15 @@ lawgroup_governance_principles = {
 lawgroup_distribution_of_power = {
 	law_group_category = power_structure
 	base_enactment_days = 150
-	enactment_approval_mult = 1.2
+	enactment_approval_mult = 1.2 #1
```

Comment:
In combination with the defines changes this means that the approval impact is 7 instead of 8 for this lawgroup, is this on purpose?